--- /data/vision/torralba/frames/data_acquisition/SyntheticStories/MultiAgent/tshu/vh_multiagent_models/models/base_nets.py
+++ /data/vision/torralba/frames/data_acquisition/SyntheticStories/MultiAgent/tshu/vh_multiagent_models/models/base_nets.py
@@ -24,9 +24,7 @@
         object_location = self.combine_obj_loc(obj_loc)
 
         num_preds = mask_goal_pred.sum(-1)
-        #norm_mask = (mask_goal_pred/num_preds.unsqueeze(-1)).unsqueeze(-1)
-        # Difference with low level policy
-        norm_mask = mask_goal_pred.unsqueeze(-1)
+        norm_mask = (mask_goal_pred/num_preds.unsqueeze(-1)).unsqueeze(-1)
 
         average_pred = (object_location * norm_mask).sum(1)
 